lib/onix/descriptive_detail.rb

```diff
@@ -489,7 +489,9 @@ def clil_categories
 
     def keywords
       kws=@subjects.select { |s| s.scheme_identifier.human=="Keywords" }.map { |kw| kw.heading_text }.compact
-      kws.map { |kw| kw.split(/;|,|\n/) }.flatten.map { |kw| kw.strip }
+      kws=kws.map { |kw| kw.split(/;|,|\n/) }.flatten.compact.map { |kw| kw.strip }
```

Comment:
À la place de `map {…}.flatten` on peut faire `flat_map {…}` !

Reply:
J'ai l'impression que ça ne marche pas comme prévu

["keyword1,keyword2"].map { |kw| kw.split(/;|,|\n/) }.flat_map{|k| k.strip}
NoMethodError: undefined method `strip' for ["keyword1", "keyword2"]:Array

Reply:
Oups, je n'ai pas précisé : c'est sur le premier `.map` qu'il faut le faire, celui du `split` !